docs/reference/api/storage/NVStore.md

```diff
@@ -0,0 +1,15 @@
+## NVStore
+
+NVStore is a lightweight module that stores data by keys in the internal flash for security purposes. For each item key, the NVStore module provides the ability to set the item data or get it. Newly set values are added to the flash (as in journal), with the effect of overriding the previous value for this key. The NVStore module ensures that power failures don't harm existing data during any operation.
```

Comment:
Query: What adds newly set values to the flash, and what does "as in journal" mean?

Reply:
Setting the data is achieved via the set API, and same goes for getting via the get API. 
As in journal means that newly added values are added to the end of the current data, overriding the previous value that was there for the same key.